plugins/periskop/dev/index.tsx

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createDevApp } from '@backstage/dev-utils';
+import { periskopPlugin } from '../src/plugin';
+
+createDevApp().registerPlugin(periskopPlugin).render();
```

Comment:
totally optional, and can also be done in a later, separate PR, but fleshing out this one lets you do local development while only loading your plugin - can be really convenient. Feel free to skip for now if you want